### ISSUE TITLE
Implement Dark Mode Functionality for Enhanced User Experience

### ISSUE THREAD

**Issue by om453** [tushargupta1504/Medical-Website#156]
Hii @tushargupta1504 

I have reviewed this project i'm raising this issue becuse i think dark mode feature should be add to this medical website to improve accessibility and visual comfort for users. 
Want to add a dark mode toggle button to access dark theme
![Screenshot 2024-10-03 182430](https://github.com/user-attachments/assets/e122b7fd-d25d-45c0-8072-dcf166010f9b)

Kindly assign me this issue so i can start working as soon as possible and pls do add gssoc-ext 24 , hacktoberfest-accepted in repository as well

Check list
[✔️] I am contributing under GSSoC-ext
[✔️] I am contributing under Hacktoberfest
[✔️] I follow contribution guidelines

**Comment by Hiteshsambhani1** [comment]
can you please assign me this issue @tushargupta1504

**Comment by Akanshaku** [comment]
I would like to resolve this issue pls assign me this under GSSoc'24 Extd @tushargupta1504

**Comment by om453** [comment]
@tushargupta1504  i have raised the PR kindly check it and merge it as well
